tasks/src/Reinforcement-learning/about.md

```diff
@@ -0,0 +1,9 @@
+## Use Cases
+
+### Robotics
+
+Reinforcement learning is used in robotics to enable robots to learn from experience and improve their performance over time. For example, a robot might be programmed to clean a floor by starting in one corner and moving in a straight line until it reaches the other corner. If the robot encounters an obstacle, it will learn from this experience and try a different route the next time it cleans the floor.
```

Comment:
```suggestion
Reinforcement learning is used in robotics to enable robots to learn from experience and improve their performance over time. For example, a robot might be programmed to clean a floor by starting in one corner and moving in a straight line until it reaches the other corner. If the robot encounters an obstacle, it will learn from this experience and try a different route the next time it cleans the floor.
```
Does this really apply, the robots have sensors no? I don't think RL is needed for this. I'd try not to give too much detail about it. WDYT @simoninithomas

Reply:
It depends on the setting but I would change to:

Reinforcement learning is used in robotics to enable robots to learn from experience and improve their performance over time. For example, a robot might **have the goal to clean** a floor by starting in one corner and moving in a straight line until it reaches the other corner. If the robot encounters an obstacle, it will learn from this experience and try a different route the next time it cleans the floor.